content/blog/shortcodes.md

```
@@ -226,6 +227,16 @@ The shortcodes can be customized with different arguments:
   </picture>
   ```
 
+### `color-scheme-selector`
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)